types/yargs/index.d.ts

```diff
@@ -9,7 +9,7 @@
 //                 ExE Boss <https://github.com/ExE-Boss>
 //                 Aankhen <https://github.com/Aankhen>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
+// TypeScript Version: 4.1
```

Comment:
This is incorrect, as this specifies the minimum supported **TypeScript** version (including `typesVersions`), so the correct way to do this would be to revert this change.

And since the minimum **TypeScript** version supported by **DefinitelyTyped** is currently **TypeScript 3.4**, this line can be deleted:
```suggestion
```

Reply:
I [removed version marker](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50722/commits/a0f87f1a09ea8a25b08d2c8a9822220b0f78219d) from the file in the 4.0 folder, as that's the one that had the 3.0 minimum